Bottom: fcd88b07540162340780ccb36cbd6b92160c3aa8
Top:    c9161c5501041581316848e3c82971b02f67ed86
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-07-03 16:39:01 -0500

Refresh of add-vmdaedalus-inventory

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index c1c2be69..6bd4fed8 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -13,25 +13,17 @@
       src: /tmp/ansible_variables
       dest: "{{inventory_hostname}}_ansible_variables"
 
-- name: Symlink on-disk copies of bacula archives
+- name: Bacula
   hosts: bacula
   remote_user: ansibleremote
   become: true
   become_user: root
   become_method: sudo
-  roles: 
-  - name: Run bacula-restores symlinks
+  roles:
+  - name: Create bacula-restores symlinks
     role: bacula-restores
     tags:
     - bacula_restores
-
-- name: Bacula
-  hosts: bacula
-  remote_user: ansibleremote
-  become: true
-  become_user: root
-  become_method: sudo
-  roles:
   - role: mariadb
     mariadb_open_firewall: true
     mariadb_remote_root_access: false
